src/plugins/objects/livemap.ts

```
@@ -0,0 +1,843 @@
+        continue;
+      }
+      if (currentEntry.tombstone === true && newEntry.tombstone === true) {
+        // both props are tombstoned - treat as noop, as there is no data to compare.
+        continue;
+      }
+
+      // both props exist and are not tombstoned, need to compare values with deep equals to see if it was changed
+      const valueChanged = !dequal(currentEntry.data, newEntry.data);
+      if (valueChanged) {
+        update.update[typedKey] = 'updated';
+        continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)